packages/wonder-blocks-tooltip/components/tooltip.md

```diff
@@ -0,0 +1,85 @@
+### Text anchor & text tooltip
+
+```js
+const React = require("react");
+const {Tooltip} = require("@khanacademy/wonder-blocks-tooltip");
+
+<Tooltip content={"The tooltip text"}>
+    Some text
+</Tooltip>
+```
+
+### Complex anchor & text tooltip
+
+In this example, we're no longer forcing the anchor root to be focusable, since the text input can take focus.
+
+```js
+const React = require("react");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const {Tooltip} = require("@khanacademy/wonder-blocks-tooltip");
+
+<Tooltip forceAnchorFocusivity={false} content={"The tooltip text"}>
+    <View>
+        Some text
+        <input />
+    </View>
+</Tooltip>
+```
+
+### Substring anchor in scrollable parent
+
+```js
+const React = require("react");
+const {View, Text} = require("@khanacademy/wonder-blocks-core");
+const {Tooltip} = require("@khanacademy/wonder-blocks-tooltip");
+
+<div>
+    <div style={{height: 100, overflow: "auto", border: "1px solid", margin: 10,}}>
+        <div style={{height: "200vh"}}>
+            <Text>This is a big long piece of text with a </Text>
```

Comment:
Please a specific typography component.  I'm planning to hide or change `Text` since it doesn't match React Native's `Text` component so we should avoid use it.

Reply:
Again, these are all changing in one of the other PRs, so I would prefer to leave as-is here, and update there.